### DISCUSSION TITLE
Nick Beattie: Rainbow Wallet

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#92] [Announcements]
## Background:

[Twitter](https://twitter.com/nickbytes)
[Website](https://www.nickbytes.com)

## Intro

Andrew: Hello, welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin. 

Justin: Hey everyone! Our guest today is Nick Beattie, a software engineer at Rainbow. Nick would you like to tell our audience a little more about yourself?

## Questions

- [x] What is rainbow wallet?
- [x] What is the sdk? What kind of things can you do with it?
- [x] What is decentralized identity? Why is it something I would want as a user?
- [x] What's some of the coolest experiences you've seen built on rainbow wallet?
- [x] How different is web3 dev than traditional dev? Is it vastly more complex, or is approachable enough for juniors to ramp up? Are the learning resources there or is it a lot of trail and error?
- [x] What features are on your roadmap that you're most excited about?
- [x] How does rainbow wallet plan to become a business? 
- [x] Do you think web3 will change how all apps are built some day?
- [x] What cool use cases do you see for smart contracts and tokens? What are some real world examples that you can point to?
- [x] What do you think the future of cryto looks like?

## Tooltips

Add them in the comments below!

## Outro

Andrew: Well that’s it for this week’s episode of devtools.fm. Be sure to follow us on YouTube and wherever you consume your podcasts. Thanks for listening!

**Comment by hipstersmoothie** [comment]
My Tooltips:

https://github.com/kuprel/min-dalle
https://twitter.com/shuding_/status/1475916082875666441

**Comment by just-be-dev** [comment]
Tooltips:

- [Haptic smart knob project](https://www.youtube.com/watch?v=Q76dMggUH1M)
- [Automatic CNC tool head changer](https://www.youtube.com/watch?v=ZeOUdosoEBM)

**Comment by nickbytes** [comment]
> What's some of the coolest experiences you've seen built on rainbow wallet?

Would it be alright if I also include cool experiences built on top of [RainbowKit](http://rainbowkit.com/)?

**Reply by just-be-dev** [reply]
Yeah, for sure

**Comment by nickbytes** [comment]
Tooltips:

- [Foundry](https://github.com/foundry-rs/foundry) is a blazing fast, portable and modular toolkit for Ethereum application development written in Rust.
- [Monome Norns Shield](https://monome.org/docs/norns/shield/) is a DIY circuit which extends a normal Raspberry Pi, turning it into a norns.
